ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -21,7 +21,7 @@
         // Using ev here so that it isn't the same as Event that we bind in the <body>
         // Works fine if we use Event, but VS will yell at you with an error in the IDE
         var ev = (Model as ServerCore.ModelBases.EventSpecificPageModel)?.Event;
-        if (ev != null && (ev.UrlString == "pd2019" || ev.UrlString == "pd2020" || ev.UrlString == "nipd2020" || ev.UrlString == "pd2021" || ev.UrlString == "nipd2021"))
+        if (ev != null && (ev.UrlString == "pd2019" || ev.UrlString == "pd2020" || ev.UrlString == "nipd2020" || ev.UrlString == "pd2021" || ev.UrlString == "nipd2021" || ev.UrlString == "pd2021.2203"))

```

Comment:
This line is getting ridiculous lol. Not asking you to fix unless you really really want to.

Reply:
Hehe, I didn't make it, but I'm definitely making it worse!